db/models/__init__.py

```diff
@@ -124,4 +124,5 @@ def projection() -> dict:
 class InterfaceDetailDoc(DocumentBaseModel):
     """InterfaceDetail DB Document Model."""
 
-    available_vlans: List[int]
+    available_tags: Dict[str, List[List[int]]]
```

Comment:
I am going to change everything to string and avoid the conversion of the key.

Reply:
Excellent. That'll be great. But just to be clear we're talking about `"vlan"` and not just `"1"` right? 

Since you're at it, please let's also make sure `mef_eline` tag_type https://github.com/kytos-ng/mef_eline/blob/master/openapi.yml#L688-L690 API payload will follow the same new pattern. We also want to still be able to load old circuits, so we'll need script, it shouldn't be that difficult but let's also cover it.

Reply:
Changed it to string. Commit [f4c0cf85727a936b497c1bde9bd5fffd35d7cec7](https://github.com/kytos-ng/topology/pull/160/commits/f4c0cf85727a936b497c1bde9bd5fffd35d7cec7)